src/components/VideoPlayer/index.js

```diff
@@ -171,6 +174,78 @@ export default class VideoPlayer extends PureComponent {
 
   resumeVideo = () => { this.video.play() }
 
+  handleKeyDown = (e) => {
+    const { target, key } = e
+    const selectingVolume = target.className.indexOf('vjs-volume-bar') > -1 ||
```

Comment:
just an FYI, another way to do this is `target.classList.contains('vjs-volume-bar')`